light/store/db/db_test.go

```diff
@@ -149,18 +149,25 @@ func Test_Concurrency(t *testing.T) {
 		go func(i int64) {
 			defer wg.Done()
 
-			dbStore.SaveSignedHeaderAndValidatorSet(
+			err := dbStore.SaveSignedHeaderAndValidatorSet(
 				&types.SignedHeader{Header: &types.Header{Height: i}}, vals)
-
-			dbStore.SignedHeader(i)
-			dbStore.ValidatorSet(i)
-			dbStore.LastSignedHeaderHeight()
-			dbStore.FirstSignedHeaderHeight()
-
-			dbStore.Prune(2)
+			require.NoError(t, err)
+
+			_, err = dbStore.SignedHeader(i)
+			require.NoError(t, err)
+			_, err = dbStore.ValidatorSet(i)
+			require.NoError(t, err)
+			_, err = dbStore.LastSignedHeaderHeight()
+			require.NoError(t, err)
+			_, err = dbStore.FirstSignedHeaderHeight()
+			require.NoError(t, err)
+
+			err = dbStore.Prune(2)
+			require.NoError(t, err)
 			_ = dbStore.Size()
 
-			dbStore.DeleteSignedHeaderAndValidatorSet(1)
+			err = dbStore.DeleteSignedHeaderAndValidatorSet(1)
+			require.NoError(t, err)
```

Comment:
I think this will fail non-deterministically.. this test was not created to check for errors

please use `_ = `

Reply:
should we have a test that is non-deterministic?

Reply:
it checks concurrency

Reply:
will make it log on error instead of failure so we can just see it fail